### ISSUE TITLE
Do: Couldn't parse this if expression

### ISSUE THREAD

**Issue by Yanpas** [koalaman/shellcheck#1809]
#### For bugs
- Rule Id (if any, e.g. SC1000): 
- My shellcheck version (`shellcheck --version` or "online"): online
- [ ] The rule's wiki page does not already cover this (e.g. https://shellcheck.net/wiki/SC2086)
- [ x] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh
if true; then
    Do the job
fi
```

#### Here's what shellcheck currently says:
![image](https://user-images.githubusercontent.com/7345761/73273972-d8e68380-41f5-11ea-9b52-8b4c2108d27a.png)

**Comment by gromgit** [comment]
Probably related to #1801 , in that shellcheck seems to treat `Do` as a reserved word, despite not meeting the criteria I highlighted in the other issue.

**Comment by koalaman** [comment]
The warning is still there, but now it parses correctly so you can ignore it as you see fit. Thanks!
